src/player.cpp

```diff
@@ -2016,8 +2016,8 @@ void player::memorial( std::ostream &memorial_file, std::string epitaph )
 
     //HP
 
-    const auto limb_hp = [this, &memorial_file, &indent]( const std::string & desc,
-    const body_part bp ) {
+    const auto limb_hp =
+    [this, &memorial_file, &indent]( const std::string & desc, const body_part bp ) {
```

Comment:
Not quite optimal, but definitly better than before. Astyle really seems to be unable to work with lambda funcions.